components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -44,6 +44,10 @@ export class LayoutFacadeService {
         });
     }
 
+    hasGlobalNotifications(): boolean {
+        return this.headerHeight === '110px';
```

Comment:
Reviewing after the fact:
This is rather brittle. Is it possible to check for an element rather than a specific (magic) size? If not, could you at least add a comment explaining the reason for this magic number?